draft-ietf-quic-multipath.md

```diff
@@ -317,6 +317,9 @@ Path ID 0.
 After the handshake concluded if negotiation of multipath support succeeded,
 endpoints SHOULD use MP_ACK frames instead of ACK frames,
 also for acknowledging so far unacknowledged 0-RTT packets, using
+Path ID 0. Respectively after a successful handsaheke, endpoints SHOULD also use
```

Comment:
Another strange use of respectively. I think for this one case just short the sentence: 
Endpoints SHOULD also use
the MP_NEW_CONNECTION_ID frame to provide new connection IDs and,
the MP_RETIRE_CONNECTION_ID frame to retire connection IDs, respectively for Path ID 0.